casper/src/main/scala/coop/rchain/casper/Validate.scala

```diff
@@ -423,12 +423,13 @@ object Validate {
     */
   def parents[F[_]: Monad: Log: BlockStore](
       b: BlockMessage,
+      genesis: BlockMessage,
```

Comment:
wouldn't it make sense to start passing `genesisHash` instead of the whole genesis block?